apps/site/styles/locals.css

```
@@ -0,0 +1,3 @@
+html[lang='ko'] {
```

```suggestion
/* Comment to explain why, this tile for ko */
html[lang='ko'] {
```

Can you document why, you put theses style for Korean

---

Comments have been added!